### DISCUSSION TITLE
Navigate and Engage with Locations

### DISCUSSION THREAD

**Discussion by ap0l0** [shine-jayakumar/insta-likecom-bot#99] [Ideas]
Hi @shine-jayakumar 

Another way I found to have endless posts is accessing "locations". The way to do this is by following these steps:

1. Open https://www.instagram.com/explore/locations/
2. Select Country --> Then City --> Then Location

The best way is asking the user to pass the full URL as parameter:

`script.py --location https://www.instagram.com/explore/locations/212962809/houston-texas/ -np 500 -lc 2
`

The user can get this URL by visiting this link: `https://www.instagram.com/explore/locations/ `

**Comment by shine-jayakumar** [comment]
Great find! I've never used this feature though. User can provide `212962809/houston-texas/` part of the URL. Rest of that remains the same.

**Comment by ap0l0** [comment]
Just the ID seems to be enough:

https://www.instagram.com/explore/locations/212962809/

**Reply by shine-jayakumar** [reply]
Even better!
